### PR TITLE
Call a method to set imf_context in platform view channel

### PR DESCRIPTION
**Pull request by seungsoo47**
* Webview which inherits from platform view uses the imf_context of engine.

### CODE REVIEW

File: shell/platform/tizen/channels/platform_view_channel.cc [hunk 1]
```diff
@@ -151,10 +156,28 @@ void PlatformViewChannel::HandleMethodCall(
         channel_->InvokeMethod("viewFocused", std::move(id));
       }
 
+      if (engine_ && engine_->text_input_channel) {
+        Ecore_IMF_Context* context =
+            engine_->text_input_channel->GetImfContext();
```
Comment (bbrto21) [inline shell/platform/tizen/channels/platform_view_channel.cc:161]: Could you simply describe how you plan to handle this imf context event?
As you already know, this context already had handlers for input widget, not web-view.

Comment (seungsoo47) [reply shell/platform/tizen/channels/platform_view_channel.cc:161]: You are right. First of all, we have to check whether the webview is one of the flutter widgets. Then, we have to make the webivew to send the engine the events like setClient or clearClient. However, I have no idea how to solve this issue yet.

Comment (bbrto21) [reply shell/platform/tizen/channels/platform_view_channel.cc:161]: All right. I don't have a specific idea either. Let's keep moving forward

**Review by bbrto21 — APPROVED** [review]
LGTM

File: shell/platform/tizen/channels/platform_view_channel.cc [hunk 2]
```diff
@@ -191,17 +213,12 @@ void PlatformViewChannel::HandleMethodCall(
       } else if (method == "setDirection") {
         FT_LOGD("PlatformViewChannel setDirection");
         result->NotImplemented();
-      } else if (method == "clearFocus") {
```
Comment (bwikbs) [inline shell/platform/tizen/channels/platform_view_channel.cc:194]: Why remove this :question:

Comment (seungsoo47) [reply shell/platform/tizen/channels/platform_view_channel.cc:194]: I did not remove it and moved just the location of the code.

Comment (bwikbs) [reply shell/platform/tizen/channels/platform_view_channel.cc:194]: Oh.. boy.  Sorry! :sweat_smile:

**Review by bwikbs — APPROVED** [review]

**Pull request merged**